### PR TITLE
Remove leftover FreeBSD related code and documentation

### PR DESCRIPTION
**Pull request by zhumo**
Removing freebsd from:
1. documentation
2. CMakelists
3. freebsd .tables and .cpps
4. generators

### CODE REVIEW

**Review by directionless — COMMENTED** [review]
Hrm. This is a good question. In the past, our theory had been that someone might revisit this, so we should leave the code in place. But, it's not wrong to re-evaluate it, and simpler code has some advantages. I am ambivalent.

**Comment by Smjert** [comment]
> Hrm. This is a good question. In the past, our theory had been that someone might revisit this, so we should leave the code in place. But, it's not wrong to re-evaluate it, and simpler code has some advantages. I am ambivalent.

I vote to remove it for now; the code will always exists in the git history for reference.

There hasn't been a direct interest in it (and with direct I mean a contributor with the interest, ability and time to support it), and we are not updating it/keeping it working since the start of 4.x, so it's just dead code.

**Comment by Smjert** [comment]
> Hrm. This is a good question. In the past, our theory had been that someone might revisit this, so we should leave the code in place. But, it's not wrong to re-evaluate it, and simpler code has some advantages. I am ambivalent.

I vote to remove it for now; the code will always exists in the git history for reference.

There hasn't been a direct interest in it (and with direct I mean a contributor with the interest, ability and time to support it), and we are not updating it/keeping it working since the start of 4.x, so it's just dead code.

**Comment by zhumo** [comment]
In other projects, we've generally followed the idea that it will exist in git history so we can revive it there, where necessary, as Stefano is saying. Deleting code is a joy! 

That said, at the very least, we would still want to get rid of the .table files right? Happy to pare this down in some way, wherever you guys decide. But I figure I would start here by submitting the maximalist approach.

File: specs/CMakeLists.txt [hunk 1]
```diff
@@ -223,32 +221,32 @@ function(generateNativeTables)
...
-    "posix/ulimit_info.table:linux,macos,freebsd"
+    "posix/sudoers.table:linux,macos"
+    "posix/suid_bin.table:linux,macos"
+    "posix/system_controls.table:linux,macos"
+    "posix/ulimit_info.table:linux,macos"
     "posix/usb_devices.table:linux,macos"
-    "posix/user_events.table:linux,macos,freebsd"
-    "sleuthkit/device_file.table:linux,macos,freebsd,windows"
-    "sleuthkit/device_hash.table:linux,macos,freebsd,windows"
-    "sleuthkit/device_partitions.table:linux,macos,freebsd,windows"
```
Comment (zhumo) [inline specs/CMakeLists.txt:251]: Oops. just noting here for later. I shouldn't delete windows.

**Comment by directionless** [comment]
Between @Smjert and @zhumo that's two votes to remove, and none to keep. I'm onboard!

@zhumo -- you mentioned not accidentally removing windows, other than that it's seems great.

**Review by directionless — APPROVED** [review]
I think that's okay?

**Comment by zhumo** [comment]
Grr yes will fix 

Mo Zhu

File: specs/CMakeLists.txt [hunk 2]
```diff
@@ -153,11 +153,9 @@ function(generateNativeTables)
     "darwin/xprotect_entries.table:macos"
     "darwin/xprotect_meta.table:macos"
     "darwin/xprotect_reports.table:macos"
-    "linux/selinux_events.table:linux"
```
[the remark this answers is not in the document]
Comment (zhumo) [reply specs/CMakeLists.txt]: Not seeing this anymore...

[the remark this answers is not in the document]
Comment (mike-myers-tob) [reply specs/CMakeLists.txt]: Yea this is not in the diff I see

**Review by directionless — APPROVED** [review]

**Pull request merged**